examples/social/README.md

```diff
@@ -0,0 +1,133 @@
+# A Social Media Example Application
+
+This example illustrates how to use channels for cross-chain messages.
+
+For simplicity, each microchain represents one user—its owner. They can subscribe to other
+users and make text posts that get sent to their subscribers.
+
+## How it Works
+
+The application's state on every microchain contains a set of posts created by this chain
+owner, and a set of posts received from other chains that it has subscribed to. The
+received posts are indexed by timestamp, sender and index.
+
+There are `Subscribe` and `Unsubscribe` operations: If a chain owner includes these in a
+new block, they can subscribe to or unsubscribe from another chain.
+
+There is also a `Post` operation: It creates a new post and sends it to a channel, so that
+it reaches all subscribers.
+
+There are corresponding `RequestSubscribe`, `RequestUnsubscribe` and `Posts` cross-chain
+message variants that are created when these operations are handled. The first two are
+sent directly to the chain we want to subscribe to or unsubscribe from. The latter goes
+to the channel.
+
+## Usage
+
+To try it out, first setup a local network with two wallets, compile the `social` example
+and create an application with it:
+
+```bash
+./scripts/run_local.sh
+
+export LINERA_WALLET="$(realpath target/debug/wallet.json)"
+export LINERA_STORAGE="rocksdb:$(dirname "$LINERA_WALLET")/linera.db"
+export LINERA_WALLET_2="$(realpath target/debug/wallet_2.json)"
+export LINERA_STORAGE_2="rocksdb:$(dirname "$LINERA_WALLET_2")/linera_2.db"
+
+cd examples/social && cargo build --release && cd ../..
+
+target/debug/linera --wallet $LINERA_WALLET --storage $LINERA_STORAGE publish-and-create examples/target/wasm32-unknown-unknown/release/social_{contract,service}.wasm
+```
+
+This will output the new application ID, e.g.:
+
+```
+e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65010000000000000001000000e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65030000000000000000000000
+```
+
+With the `wallet show` command you can find the ID of the application creator's chain:
+
+```bash
+target/debug/linera --wallet $LINERA_WALLET --storage $LINERA_STORAGE wallet show
+```
+
+```
+e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65
+```
+
+
+Now start a node service for each wallet, using two different ports:
+
+```bash
+target/debug/linera --wallet $LINERA_WALLET --storage $LINERA_STORAGE service --port 8080
+target/debug/linera --wallet $LINERA_WALLET_2 --storage $LINERA_STORAGE_2 service --port 8081
```

Comment:
Two things here:

1. Same nit-pick as above regarding double quotes
2. Add a `&`, so that both run in the background

```suggestion
target/debug/linera --wallet "$LINERA_WALLET" --storage "$LINERA_STORAGE" service --port 8080 &
target/debug/linera --wallet "$LINERA_WALLET_2" --storage "$LINERA_STORAGE_2" service --port 8081 &
```